data/mods/MindOverMatter/overmap/overmap_terrain.json

```diff
@@ -165,6 +173,7 @@
     ],
     "name": "open air",
     "sym": ".",
+    "vision_levels": "large_building",
```

Comment:
Should this be `roof_or_air`? They'll look like a large building then disappear into air when you see them with detail.

Reply:
You're right, thank you for pointing it out.